eng/pipelines/templates/steps/build.yml

```diff
@@ -47,7 +41,7 @@ steps:
 
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
```

Comment:
This is really the main thrust of the PR. `PackageInfo` folder becomes the set of targeted packages, versus honoring a hardcoded `Artifacts` list. The hardcoded artifacts will still be utilized in the `release` phases of the `internal` build definition.